web/client/utils/FeatureGridUtils.js

```diff
@@ -162,19 +162,50 @@ module.exports = {
     applyAllChanges: (orig, changes = {}) => applyChanges(orig, changes[orig.id] || {}),
     applyChanges,
     gridUpdateToQueryUpdate: ({attribute, operator, value, type} = {}, oldFilterObj = {}) => {
+        const andGroup = {
+            id: 9999, // dummy number representing group id
+            logic: "AND",
+            index: 0
+        };
+
+        let mergeFilters = oldFilterObj.groupFields 
+            ? (oldFilterObj.groupFields.filter(obj => obj.id === 1 ).length > 0 && oldFilterObj.filterFields.length > 0 
+               ? true // checks if query panel filter is active
+                : oldFilterObj.filterFields.length - 1 > 0 
```

Comment:
2.  In another situation Query panel filter is active when there are more than 1 filterfields in oldFilterObj or when the the first object in filterFields array has loading property.